course/src/main/java/org/openedx/course/presentation/container/CourseContainerViewModel.kt

```diff
@@ -769,4 +772,14 @@ class CourseContainerViewModel(
             }
         )
     }
+
+    fun requestNotificationPermission(
+        activity: FragmentActivity,
+        permissionLauncher: ActivityResultLauncher<String>,
+    ) {
+        pushManager.requestNotificationPermission(
```

Comment:
Won't this make it dependent on the Notifications Module, as we aren't providing any other implementation for it?

Reply:
moved `PermissionUtils` to the core module.